### PR TITLE
fix: 🐛 Adapted Editor List Styles

### LINKED ISSUE #3492: 🐛 [Bug] Lists in posts and comments are not displayed as lists
> [sushidave](https://github.com/sushidave) **Authored by [sushidave](https://github.com/sushidave)**
_<time datetime="2020-04-20T19:25:09Z" title="Monday, April 20th 2020, 9:25:09 pm +02:00">Apr 20, 2020</time>_

---

## :bug: Bugreport

Since release 0.6.0 numbered and unordered lists in posts and comments are no longer displayed as lists anymore, nor in the editor.

hc-bug-3492-screenshot

### Steps to reproduce the behavior
1. Create a new post or a comment.
2. Format any text by clicking one of the list buttons of the toolbar.
3. The list is not displayed as a list.
4. Save the post or comment => the list is not displayed as a list.

### Expected behavior

The lists are displayed as lists.

### Version & Environment
 Type: [Notebook]      
 - OS: [macOS]      
 - Browser: [Firefox] 
 - Version [75.0]  
 - Device: [MacBook]  

### Additional context

Checked with @mattwr18

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Adaption of PR #3493 .

### Issues

- fixes #3492
- close #3493

### Todo

- [x] None

### CODE REVIEW

**Comment by Tirokk** [comment]
The CSS could be done better and more general by a CSS expert.
Especially it would be nice to get it working with `list-style-position: outside;`, which I couldn’t get to work and had to work around.

But this will do it for now and spend not much more time into it. @Mogge @ulfgebhardt

**Review by ulfgebhardt — APPROVED** [review]
Looks clean. Feel free to merge

**Review by Mogge — DISMISSED** [review]
Thank you for pick this up. Why do we have changes in `yarn.lock`?

File: yarn.lock [hunk 1]
```diff
@@ -5107,7 +5107,7 @@ request-progress@3.0.0:
   dependencies:
     throttleit "^1.0.0"
 
-"request@github:cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16":
+request@cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16:
```
Comment (Mogge) [inline yarn.lock:5110]: Why?

Comment (Tirokk) [reply yarn.lock:5110]: It came in by `yarn install` after I cloned the repo absolutely fresh in a new folder.
This was not on purpose. It’s was automatically. So what can or should I do? @Mogge

Comment (Tirokk) [reply yarn.lock:5110]: I investigated it further:

The unusual notation at

https://github.com/Ocelot-Social-Community/Ocelot-Social/blame/master/yarn.lock#L5110

changed now to the new one was already in the master by a Cypress bump 👆🏼 11 months ago.
The new one locks more correct to me if I compare to the other entries in `yarn.lock` …

Comment (Mogge) [reply yarn.lock:5110]: This is line 5110 in master:
```
"request@github:cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16":
```
Why change it with this PR? Do as you like, but I do not like it, to change `yarn.lock'` in a PR that does not touch any dependencies.

Comment (Tirokk) [reply yarn.lock:5110]: So you like me to put things like this in a separate PR? @Mogge

**Pull request merged**